gfx_direct3d_common.h

```diff
@@ -21,6 +21,7 @@ struct CCFeatures {
 };
 
 void ThrowIfFailed(HRESULT res);
```

Comment:
I believe this line should be deleted since you add the line below it.

Reply:
Both overloads of ThrowIfFailed are used. Direct3D 12 renderer uses the first exclusively.